Assets/Tests/PlayModeTests/EstimateTickTest/EstimateTestStraightPath.cs

```diff
@@ -62,23 +61,24 @@ public void ClearSimulator()
         [Test(ExpectedResult = null)]
         public IEnumerator EstimateTicksToTile_StraightPath()
         {
+            var offset = new Vector2Int(10, 0);
+            InitializeTestingSimulator(offset, true);
             var robotCurrentPosition = _testAlgorithm.Controller.SlamMap.CoarseMap.GetCurrentPosition();
-            var targetTile = robotCurrentPosition + new Vector2Int(10, 0);
+            var targetTile = robotCurrentPosition + offset;
             var expectedEstimatedTicks = _robot.Controller.EstimateTimeToTarget(targetTile);
```

Comment:
This stuff must only be done in the algorithm

Reply:
Done